packages/apktool.vm/apktool.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>apktool.vm</id>
-    <version>2.9.0.20231024</version>
+    <version>2.9.1.20241219</version>
```

Comment:
[nitpick] The date in the forth segment is not needed as you are increasing the version

Reply:
Gotcha! I can make that change back to stay consistent.